### PR TITLE
[EAGLE-4112] - yolox

### PR DESCRIPTION
**Pull request by phatvo9**
### What
Add yolox

### CODE REVIEW

**Comment by ackizilkale** [comment]
I am in the model_upload/visual_detector folder. Did wget, pip install. 

When I do `clarifai build model yolox` I get `FileNotFoundError: [Errno 2] No such file or directory: 'pytest -s --log-level=INFO yolox/test.py'`

**Comment by phatvo9** [comment]
> I am in the model_upload/visual_detector folder. Did wget, pip install.
> 
> When I do `clarifai build model yolox` I get `FileNotFoundError: [Errno 2] No such file or directory: 'pytest -s --log-level=INFO yolox/test.py'`

It's an OS issue when running subprocess, fixed in [#305 in clarifai-python](https://github.com/Clarifai/clarifai-python/pull/305)

**Review by ackizilkale — APPROVED** [review]

File: model_upload/visual_detector/Readme.md [hunk 1]
```diff
@@ -48,6 +48,23 @@ Install dependecies to test locally
...
 ```
 
+## [YOLOX](./yolox/)
+
+[YOLOX](https://github.com/open-mmlab/mmdetection/tree/v3.0.0rc3/configs/yolox) Requirements to run tests locally:
+
+Download checkpoint and save it in `yolofx/configs/yolox/`, e.g download `x` type of model:
+
+```bash
+$ wget -P yolofx/configs/yolox/ https://download.openmmlab.com/mmdetection/v2.0/yolox/yolox_x_8x8_300e_coco/yolox_x_8x8_300e_coco_20211126_140254-1ef88d67.pth
```
Comment (ackizilkale) [inline model_upload/visual_detector/Readme.md:58]: correct yolofx spelling

**Comment by ackizilkale** [comment]
one very minor spelling mistake to fix. Other than works fine.

**Pull request merged**